src/test/perl/rbe_rule_parser.t

```diff
@@ -350,7 +406,7 @@ set_file_contents($DPM_CONF_FILE,$DPM_INITIAL_CONF_1);
 
 # Test  simple variable substitution
 set_file_contents($DPM_CONF_FILE,$DPM_INITIAL_CONF_1);
```

Comment:
put all this in a function instead of copy/pasting, like
```perl
sub test1 
{
    my ($fn, $data, $args,$expected, $msg);
    set_file_contents($fn,$data);
    my $fh = CAF::RuleBasedEditor->open($fn, log => $obj);
    ok(defined($fh), "$fn was opened $msg");
     $changes = $fh->updateFile(@$args);
    is("$fh", $expected, "$fn has expected contents $msg");
    $fh->close();
}
test1($DPM_CONF_FILE, $DPM_INITIAL_CONF_1, [\%dpm_config_rules_1, $dpm_options], $DPM_EXPECTED_CONF_1,"(config 1)");
```